docs/reference/searchable-snapshots/index.asciidoc

```diff
@@ -155,27 +155,46 @@ for search. Many searches will need to retrieve only a small subset of the
 total shard data before returning results.
 
 To mount a searchable snapshot index with the shared cache mount option, you
-must configure the `xpack.searchable.snapshot.shared_cache.size` setting to
-reserve space for the cache on one or more nodes. Indices mounted with the
-shared cache mount option are only allocated to nodes that have this setting
-configured.
+must have one or more nodes with a shared cache available. By default,
+dedicated frozen data tier nodes (nodes with the `data_frozen` role and no other
+data roles) have a shared cache configured using the greater of 90% of total
+disk space and total disk space subtracted a headroom of 100GB.
+
+Using a dedicated frozen tier is highly recommended for production use. If you
+do not have a dedicated frozen tier, you must configure the
+`xpack.searchable.snapshot.shared_cache.size` setting to reserve space for the
+cache on one or more nodes. Indices mounted with the shared cache mount option
+are only allocated to nodes that have a shared cache.
 
 [[searchable-snapshots-shared-cache]]
 `xpack.searchable.snapshot.shared_cache.size`::
+(<<static-cluster-setting,Static>>)
+The size of the space reserved for the shared cache, either specified as a
+percentage of total disk space or an absolute <<byte-units,byte value>>.
+Defaults to 90% of total disk space on dedicated frozen data tier nodes,
+otherwise `0b`.
+
+`xpack.searchable.snapshot.shared_cache.size.max_headroom`::
 (<<static-cluster-setting,Static>>, <<byte-units,byte value>>)
-The size of the space reserved for the shared cache. Defaults to `0b`, meaning
-that the node has no shared cache.
+For dedicated frozen tier nodes, the max headroom to maintain. Defaults to 100GB
+on dedicated frozen tier nodes when
+`xpack.searchable.snapshot.shared_cache.size` is not explicitly set, otherwise
+-1 (not set). Can only be set when `xpack.searchable.snapshot.shared_cache.size`
+is set as a percentage.
 
-You can configure the setting in `elasticsearch.yml`:
+You can configure the settings in `elasticsearch.yml`:
 
 [source,yaml]
 ----
 xpack.searchable.snapshot.shared_cache.size: 4TB
 ----
 
-IMPORTANT: You can only configure this setting on nodes with the
+IMPORTANT: You can only configure these settings on nodes with the
 <<data-frozen-node,`data_frozen`>> role.
 
+IMPORTANT: You can only configure these settings on nodes that use a single
```

Comment:
Perhaps more of a question to the docs team: I wonder if we should have two "important" notices following each other, or whether it should be all folded into one.

Reply:
I tried to find other places with two, but found none and did find some where it could have been split into two. So I collapsed it to one in 1b8ede452ae11c6f16eea393dd145b6756b324d2